x/leveragelp/keeper/query_position.go

```diff
@@ -21,12 +21,19 @@ func (k Keeper) Position(goCtx context.Context, req *types.PositionRequest) (*ty
 	if err != nil {
 		return nil, err
 	}
+	updatedLeveragePosition := types.QueryPosition{}
+
+	updated_leverage := position.LeveragedLpAmount.Quo(position.LeveragedLpAmount.Sub(position.Liabilities))
```

Comment:
`It should display `LP value in usdc/ ( LP value in USDC - Liability in usdc)``

LeveragedLpAmount should be in usdc